docs/target-filters.rst

```diff
@@ -3,14 +3,15 @@
 Target filters
 ==============
 
-By default, all records are made available to all users.
+By default all records in a collection are made available.  However, there are some use cases where it is more practical to use a single server side collection and filter record visibility in the browser.  Target filters allows for use cases.
```

Comment:
*allows for use cases* ? Is this a typo or a grammar construction I don't know?

Reply:
Thank you.  Fixed.